### ISSUE TITLE
undefined error in couch_mrview_changes_since_tests:test_compact/1

### ISSUE THREAD

**Issue by nickva** [apache/couchdb#685]
```
 changes_since tests

    couch_mrview_changes_since_tests:109: test_basic...ok

    couch_mrview_changes_since_tests:132: test_basic_since...ok

    couch_mrview_changes_since_tests:145: test_basic_count...ok

    couch_mrview_changes_since_tests:154: test_basic_count_since...ok

    undefined

    *** instantiation of subtests failed ***

**in function gen_server:call/3 (gen_server.erl, line 188)

in call from couch_mrview_util:get_view_index_state/5 (src/couch_mrview_util.erl, line 83)

in call from couch_mrview_util:get_view/4 (src/couch_mrview_util.erl, line 45)

in call from couch_mrview:count_view_changes_since/5 (src/couch_mrview.erl, line 304)

in call from couch_mrview_changes_since_tests:test_compact/1 (test/couch_mrview_changes_since_tests.erl, line 163)

**exit:{unknown_info,{gen_server,call,[<0.23284.1>,{get_state,11},infinity]}}

```

https://couchdb-vm2.apache.org/ci_errorlogs/travis-couchdb-254596045-2017-07-17T20%3A28%3A00.475245/couchlog.tar.gz

**Comment by davisp** [comment]
@nickva I'm pretty sure this is a duplicate of #649 which was fixed by the PR #667. I went back and verified that the commit from the run you linked in the description does not include that fix.

**Comment by nickva** [comment]
Good catch. Let's close it then.
